receiver/snmpreceiver/config.go

```diff
@@ -47,7 +47,10 @@ var (
 	errMsgColumnAttributeBadName           = `metric '%s' column_oid attribute name '%s' must match an attribute config`
 	errMsgColumnAttributeBadValue          = `metric '%s' column_oid attribute '%s' value '%s' must match one of the possible enum values for the attribute config`
 	errMsgColumnResourceAttributeBadName   = `metric '%s' column_oid resource_attribute '%s' must match a resource_attribute config`
-	errMsgColumnIndexedAttributeRequired   = `metric '%s' column_oid must either have a resource_attribute or an indexed_value_prefix/oid attribute`
+	errMsgColumnIndexedIdentifierRequired   = `metric '%s' column_oid must either have an indexed resource_attribute or an indexed_value_prefix/oid attribute`
+	errMsgMultipleKeysSetOnResourceAttribute = `resource attribute %s has too many keys set, must have only one of oid, scalar_oid, or indexed_value_prefix`
```

Comment:
```suggestion
	errMsgMultipleKeysSetOnResourceAttribute = `resource attribute %s must have only one of oid, scalar_oid, or indexed_value_prefix`
```

The notion of keys may be a bit abstract to users. I think it's ok to just state the requirement like this.